app/src/productDetailsController.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Created by Krisztián on 2016.12.09..
+ */
+
+(function () {
+    angular
+        .module('webshopModule')
+        .controller('ProductDetailsCtrl', ProductDetailsController);
+
+    function ProductDetailsController($http, $scope) {
+            $http({
+                method: 'GET',
+                url: 'http://localhost:1337/product/{id}'
```

Comment:
We have to provide an explicit `id` to the backend. Please check out the `$routeParams` service.